calculator.rb

```diff
@@ -0,0 +1,63 @@
+#combine all if blocks into one method
+def process_operator(operation, numbers)
+  if operation == "add" || operation == "+"
+    return do_math("+", numbers[0], numbers[1])
+  elsif operation == "subtract" || operation == "-"
+    return do_math("-", numbers[0], numbers[1])
+  elsif operation == "multiply" || operation == "*"
+    return do_math("*", numbers[0], numbers[1])
+  elsif operation == "divide" || operation == "/"
+    return do_math("/", numbers[0], numbers[1])
+  elsif operation == "modulo" || operation == "%"
+    return do_math("%", numbers[0], numbers[1])
+  end
```

Comment:
If the user inputs a bogus operator, your program quits silently. Instead, it would be polite to let them know what went wrong.